investpy/__init__.py

```diff
@@ -2254,9 +2432,11 @@ def get_etf_historical_data(etf, country, from_date, to_date, as_json=False, ord
         req = requests.post(url, headers=head, data=params)
```

Comment:
All the changes done not related with `cross_currencies` should not be included since the previous code was properly formatted and the published changes are just modifications on how the data is tabulated or presented to the programmer. So **just the additions related to cross currencies data retrieval will be taken into consideration**, unless changes in any other function improves it.